src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,19 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const { user, completed, title } = todo;
+
+  return (
+    <article className={
+      classNames(
+        'TodoInfo',
+        { 'TodoInfo--completed': completed },
+      )}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+      <UserInfo user={user} />
```

Comment:
![image](https://github.com/mate-academy/react_static-list-of-todos-js/assets/70009205/7333fa03-6323-425f-900d-e78553284144)

```suggestion
      {user && <UserInfo user={user} />}
```